test/index.js

```diff
@@ -2942,7 +2969,7 @@ describe('UnaryType', function() {
   it('is a quaternary function', function() {
     eq(typeof $.UnaryType, 'function');
     eq($.UnaryType.length, 4);
-    eq($.UnaryType.toString(), 'UnaryType :: String -> String -> (Any -> Boolean) -> (t a -> Array a) -> (Type -> Type)');
+    eq($.UnaryType.toString(), 'UnaryType :: String -> String -> (Any -> Boolean) -> (t a -> Array a) -> Function');
```

Comment:
This is the result of changing the return type from `Function_([Type, Type])` to `AnyFunction`. Why make this change? `$.UnaryType` returns a function which *itself* performs type checking. Leaving type checking to the inner function results in clearer error messages. Consider, for example, the treatment of `$.Array(0)`:

```diff
-UnaryType :: String -> String -> (Any -> Boolean) -> (t a -> Array a) -> (Type -> Type)
-                                                                          ^^^^
-                                                                           1
+Array :: Type -> Type
+         ^^^^
+          1
```